tests/validation/tests/v3_api/test_v2_cis_scan.py

```diff
@@ -68,10 +78,203 @@ def test_install_v2_cis_benchmark():
     install_v2_app(client, rancher_repo, cis_charts, CHART_NAME, ns)
 
 
+def test_v2_create_clusterscan_permissive():
+    """
+    Create cluster scan with profile: "rke-profile-permissive"
+    Validate the scan CR and the scan report generated
+    """
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # install CIS if not installed
+    if not check_v2_app_installed(client,CHART_NAME):
+        install_cis_v2(client, cluster_id, cluster_name, rancher_repo)
+
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    # create cluster scans
+    scan_profile = "rke-profile-permissive"
+    scan = create_clusterscan(client, scan_profile)
+
+    # validate scan CR
+    benchmark_version = cis_scan_profile_benchmark[scan_profile]
+    validate_clusterscan_cr_and_report(
+        client, scan, scan_profile, benchmark_version,
+        cis_scan_report_results[benchmark_version]["total"],
+        cis_scan_report_results[benchmark_version]["pass"],
+        cis_scan_report_results[benchmark_version]["fail"],
+        cis_scan_report_results[benchmark_version]["notApplicable"],
+        cis_scan_report_results[benchmark_version]["skip"])
+
+
+def test_v2_create_clusterscan_default():
+    """
+        Create cluster scan with profile: ""
+        Default profile for the cluster will be picked.
+        Currently testing it on an RKE cluster so the default scan profile
+        will be "rke-profile-permissive"
+        Validate the scan CR and the scan report generated
+        """
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # check if CIS is already installed and uninstall the app
+    if not check_v2_app_installed(client, CHART_NAME):
+        install_cis_v2(client, cluster_id, cluster_name, rancher_repo)
+
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    # create cluster scans
+    scan_profile = ""
+    scan = create_clusterscan(client, scan_profile)
+
+    if cluster_detail["cluster"]["provider"] == "rke":
+        scan_profile = "rke-profile-permissive"
+    benchmark_version = cis_scan_profile_benchmark[scan_profile]
```

Comment:
What will happen if the provider is not rke? Will this test report a failure?

Reply:
Good question. I will write some code up for when its not rke.